org.adempiere.base/src/org/compiere/model/MTable.java

```diff
@@ -722,9 +722,9 @@ protected boolean afterSave (boolean newRecord, boolean success)
 			return success;
 		//	Sync Table ID
 		MSequence seq = MSequence.get(getCtx(), getTableName(), get_TrxName());

```

Comment:
The whole MSequence seq block should be enclosed inside a !isView() block (why call MSequence.get if it is a view ?).